ledger/apply/keyreg.go

```diff
@@ -67,6 +66,7 @@ func Keyreg(keyreg transactions.KeyregTxnFields, header transactions.Header, bal
 		record.VoteFirstValid = 0
 		record.VoteLastValid = 0
 		record.VoteKeyDilution = 0
+		record.IncentiveEligible = false
```

Comment:
I think `IncentiveEligible` ought to remain unchanged when an account goes offline gracefully with a `keyreg`.  The point of the extra fee is a penalty for getting suspended. If someone properly goes offline, they should be able to go back online later without the extra charge.